packages/flutter/test/painting/rounded_rectangle_border_test.dart

```diff
@@ -97,28 +97,28 @@ void main() {
 
     expect(
       ShapeBorder.lerp(r, c, 0.1).toString(),
-      'RoundedRectangleBorder(BorderSide(Color(0xff000000), 0.0, BorderStyle.none), BorderRadius.circular(10.0), 10.0% of the way to being a CircleBorder)',
+      equalsIgnoringHashCodes('RoundedRectangleBorder(BorderSide(width: 0.0, style: none), BorderRadius.circular(10.0), 10.0% of the way to being a CircleBorder)'),
```

Comment:
These don't need to use `equalsIgnoringHashCodes` anymore.

BTW, I did land a change that allows `equalsIgnoringHashCodes` to take a List<String> or a string.